plugins/telemetry-otel/src/main/java/org/opensearch/telemetry/tracing/exporter/OtlpGrpcSpanExporterProvider.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.telemetry.tracing.exporter;
+
+import org.opensearch.common.settings.Settings;
+import org.opensearch.telemetry.OTelTelemetrySettings;
+
+import java.util.Collection;
+
+import io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporter;
+import io.opentelemetry.sdk.common.CompletableResultCode;
+import io.opentelemetry.sdk.trace.data.SpanData;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+
+/**
+* OtlpGrpcSpanExporterProvider class creates an instance of OtlpGrpcSpanExporter
+*/
+public final class OtlpGrpcSpanExporterProvider implements SpanExporter {
+
+    private OtlpGrpcSpanExporter delegate;
+
+    private OtlpGrpcSpanExporterProvider(OtlpGrpcSpanExporter exporter) {
+        this.delegate = exporter;
+    }
+
+    /**
+     * create method is expected by OTelSpanExporterFactory. This creates an instance of
+     * OtlpGrpcSpanExporter and sets https endpoint.
+     * @param settings settings
+     * @return OtlpGrpcSpanExporterProvider instance.
+     */
+    public static OtlpGrpcSpanExporterProvider create(Settings settings) {
+        OtlpGrpcSpanExporter exporter;
+        String endpoint = OTelTelemetrySettings.TRACER_SPAN_EXPORTER_ENDPOINT.get(settings);
+        // if endpoint is empty, do not set Endpoint
+        if (endpoint.isEmpty()) {
+            exporter = OtlpGrpcSpanExporter.builder().build();
```

Comment:
what is the default endpoint used when it is not specified explicitly? Also what happens if the endpoint is down? would the traces be discarded?

Reply:
The default endpoint is "http://localhost:4317" [link](https://github.com/open-telemetry/opentelemetry-java/blob/2e2c308293800901aed771c3e56bba71b419257a/exporters/otlp/all/src/main/java/io/opentelemetry/exporter/otlp/trace/OtlpGrpcSpanExporterBuilder.java#L32).

If endpoint is down we will get these exceptions in log 
Failed to export spans. The request could not be executed. Full error message:

yes traces would be discarded,

Reply:
Please call the default endpoint explicitly.

Reply:
These spans are exported in async fashion and how would it impact the OpenSearch process in case exporter is running slow.